domains/faucet.mxnticek.is-not-a.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Faucet for DuinoCoin",
+    "domain": "is-not-a.dev",
+    "subdomain": "faucet.mxnticek",
+
+    "owner": {
+        "repo": "https://github.com/VlastikYoutubeKo/faucet-duinocoin",
+        "email": "novotny.vlastimil@odjezdy.online"
+    },
+
+    "record": {
+        "A": "38.242.156.120"
```

Comment:
Check are failing this PR is not valid, this is due to invalid JSON requirements.